lightningd/plugin.c

```diff
@@ -1009,18 +1007,15 @@ void plugins_init(struct plugins *plugins, const char *dev_plugin_debug)
 			p->timeout_timer = NULL;
 		else {
 			p->timeout_timer
-				= new_reltimer(&plugins->timers, p,
+				= new_reltimer(&plugins->ld->timers, p,
 					       time_from_sec(PLUGIN_MANIFEST_TIMEOUT),
 					       plugin_manifest_timeout, p);
 		}
 		tal_free(cmd);
 	}
 	while (plugins->pending_manifests > 0) {
```

Comment:
The `while` loop can be omitted entirely, we should probably `assert(plugins->pending_manifests == 0);` after the `io_loop_with_timers`.

Reply:
Or replace the `while` with an `if`.

Reply:
Ack, good catch.